### PR TITLE
change test-snapps to berkeley

### PR DESCRIPTION
**Pull request by QuiteStochastic**
berkeley qa testnet fixes

### CODE REVIEW

File: helm/archive-node/templates/db-bootstrap.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ spec:
       - name: import-schema
         image: bitnami/postgresql
         command: ["bash", "-c"]
-        args: ["PGPASSWORD={{ .Values.postgresql.postgresqlPassword }} psql --username {{ .Values.postgresql.postgresqlUsername }} --host {{ tpl .Values.archive.postgresHost . }} --port {{ .Values.archive.ports.postgres }} --dbname {{ .Values.archive.postgresDB}} -f <(curl -Ls {{ .Values.archive.remoteSchemaFile }})"]
+        args: ["{{- range .Values.archive.remoteSchemaAuxFiles }} curl -o {{.name}} {{.url}} {{- end }} && PGPASSWORD={{ .Values.postgresql.postgresqlPassword }} psql --username {{ .Values.postgresql.postgresqlUsername }} --host {{ tpl .Values.archive.postgresHost . }} --port {{ .Values.archive.ports.postgres }} --dbname {{ .Values.archive.postgresDB}} -f <(curl -Ls {{ .Values.archive.remoteSchemaFile }})"]
```
Comment (lk86) [inline helm/archive-node/templates/db-bootstrap.yaml:17]: I don't like it all in one line, take a look at the tutorial here for how to space out your file while still producing a one-line result: https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace

File: automation/terraform/testnets/berkeley/main.tf [hunk 2]
```diff
@@ -141,8 +143,8 @@ module "test-snapps" {
   snark_coordinators = [
     {
       snark_worker_replicas = 5
-      snark_worker_fee      = "0.025"
-      snark_worker_public_key = "B62qk4nuKn2U5kb4dnZiUwXeRNtP1LncekdAKddnd1Ze8cWZnjWpmMU"
+      snark_worker_fee      = "0.01"
+      snark_worker_public_key = "B62qmQsEHcsPUs5xdtHKjEmWqqhUPRSF2GNmdguqnNvpEZpKftPC69e"
```
Comment (lk86) [inline automation/terraform/testnets/berkeley/main.tf:147]: Is this public key in the ledger?

**Review by lk86 — APPROVED** [review]
Left a small question but it lgtm. The setup for the filename is a little hacky but so was my original proposal to hardcode it, and your comment explains how to work with it, so lgtm

File: automation/terraform/testnets/berkeley/main.tf (hunk 2 above)
Comment (QuiteStochastic) [reply automation/terraform/testnets/berkeley/main.tf:147]: yes, this was per deepthi's suggestion this morning.  this is the faucet i believe, the first account in the ledger.

File: automation/terraform/modules/o1-integration/locals.tf [hunk 3]
```diff
@@ -37,7 +37,9 @@ locals {
     postgresDB              = "archive"
     postgresqlUsername      = "postgres"
     postgresqlPassword      = "foobar"
+    # remoteSchemaFile needs to be just the script name, not a url.  remoteSchemaAuxFiles needs to be a list of urls of scripts, one of these urls needs to be the url of the main sql script that invokes the other ones.  sorry it's confusing
```
Comment (psteckler) [inline automation/terraform/modules/o1-integration/locals.tf:40]: maybe you want to rename these to `schemaScript` and `schemaScriptUrls`

**Review by psteckler — APPROVED** [review]
one comment

**Pull request merged**